MODIFIED_GIT_WORKFLOW.md

```diff
@@ -0,0 +1,35 @@
+# Modified Git Flow
+The Economist Digital Solutions Mobile Team uses [Git Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) with the [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) rather than forking.
+
+## Feature branch workflow
+The core idea behind the Feature Branch Workflow is that all feature development should take place in a dedicated branch instead of the master branch. This encapsulation makes it easy for multiple developers to work on a particular feature without disturbing the main codebase. It also means the master branch will never contain broken code, which is a huge advantage for continuous integration environments.
+
+The Git Feature Branch Workflow is a composable workflow that can be leveraged by other high-level Git workflows.
+
+Feature branches generally take the form of the jira ticket numbers. This allows Jira to record the history of the ticket and whether it has been merged and deployed.
+
+(https://github.com/sandeepasrani/ds-best-coding-practices/blob/modified-git-flow/Deployment-Git.png)
+
+## Gitflow Workflow
+Gitflow is ideally suited for projects that have a scheduled release cycle.
+
+Instead of a single master branch, this workflow uses two branches to record the history of the project. The master branch stores the official release history, and the develop branch serves as an integration branch for features. It's also convenient to tag all commits in the master branch with a version number.
+
+Each new feature should reside in its own branch, which can be pushed to the central repository for backup/collaboration. But, instead of branching off of master, feature branches use develop as their parent branch. When a feature is complete, it gets merged back into develop by (Submitting a PR)[#####Submitting PR's]. Features should never interact directly with master.
+
+#### Hotfix Branches
+Hotfix branches arises from an unplanned state when critical issues in production code needs to be addressed immediately. Hotfix branch is branched off from master and must be merge back into development and master. Note that when hotfix branch is merged into master it becomes a new (hotfix) release that is pushed to production.
+
+#### Release Branches (Optional)
+Once develop has acquired enough features for a release (or a predetermined release date is approaching), you fork / tag a release branch off of develop. Creating this branch starts the next release cycle, so no new features can be added after this point—only bug fixes, documentation generation, and other release-oriented tasks should go in this branch. Once it's ready to ship, the release branch gets merged into master and tagged with a version number. In addition, it should be merged back into develop, which may have progressed since the release was initiated.
```

Comment:
I don't quite understand this one. Do you put new code in the release branch? If not, why is it merged back to master/develop?Or is it only used for packaging and releasing to the app store? I think this needs a little more clarification and why it's optional.

Reply:
This is optional. Incase there is need for a small section of the team to do some minor modifications before release which could be a plceholder branch which is similar to hotfix branch. I am of the opinion to remove it and not do it.

Reply:
We have never actually done it either, in my last multiple years of working I think there has been only 1 scenario where we would have done that but we were a very big team of 15 of us working on an app and there were 3 guys from the team who before release had to make small UI fixes etc before release. So we seperated out their branch for the hardening sprint while the other team continued to work on develop and pull this release branch into master and develop accordingly.

Reply:
Ah ok, might be best to remove it for now as I think it's a bit more confusing then helpful...